src/registrar/management/commands/load_organization_data.py

```diff
@@ -0,0 +1,293 @@
+        )
+        self.bulk_update_domain_information(args.debug)
+
+    def load_json_settings(self, options, migration_json_filename):
+        """Parses options from the given JSON file."""
+        json_filepath = os.path.join(options["directory"], migration_json_filename)
+
+        # If a JSON was provided, use its values instead of defaults.
+        with open(json_filepath, "r") as jsonFile:
+            # load JSON object as a dictionary
+            try:
+                data = json.load(jsonFile)
+
+                skipped_fields = ["domain_additional_filename", "organization_adhoc_filename"]
+                # Iterate over the data from the JSON file. Skip any unused values.
+                for key, value in data.items():
+                    if value is not None and value.strip() != "":
+                        # If any key in skipped_fields has a value, then
+                        # we override what is specified in the JSON.
+                        if options not in skipped_fields:
+                            options[key] = value
+
+            except Exception as err:
+                logger.error(
+                    f"{TerminalColors.FAIL}"
+                    "There was an error loading "
+                    "the JSON responsible for providing filepaths."
+                    f"{TerminalColors.ENDC}"
+                )
+                raise err
+
+            return options
+
+    def prepare_update_domain_information(self, target_transition_domains: List[TransitionDomain], debug):
+        """Returns an array of DomainInformation objects with updated organization data."""
+        if len(target_transition_domains) == 0:
+            raise LoadOrganizationError(code=LoadOrganizationErrorCodes.EMPTY_TRANSITION_DOMAIN_TABLE)
+
+        # Grab each TransitionDomain we want to change.
+        transition_domains = TransitionDomain.objects.filter(
+            username__in=[item.username for item in target_transition_domains],
+            domain_name__in=[item.domain_name for item in target_transition_domains],
+        )
+
+        # This indicates that some form of data corruption happened.
+        if len(target_transition_domains) != len(transition_domains):
+            raise LoadOrganizationError(code=LoadOrganizationErrorCodes.TRANSITION_DOMAINS_NOT_FOUND)
+
+        # Maps TransitionDomain <--> DomainInformation.
+        # If any related organization fields have been updated,
+        # we can assume that they modified this information themselves - thus we should not update it.
+        domain_informations = DomainInformation.objects.filter(
+            domain__name__in=[td.domain_name for td in transition_domains],
+            address_line1__isnull=True,
+            city__isnull=True,
+            state_territory__isnull=True,
+            zipcode__isnull=True,
+        )
+        filtered_domain_informations_dict = {di.domain.name: di for di in domain_informations if di.domain is not None}
```

Comment:
This assumes that if any of those fields are modified, then a user likely had their hands on it - thus we shouldn't update.

This may be overly cautious, or it may not. It doesn't have to be this way, instead we can just do a check to not update fields that already have data in them. Thoughts?